xds/src/main/java/io/grpc/xds/XdsLoadReportClientImpl.java

```diff
@@ -334,4 +335,22 @@ private void cleanUp() {
       }
     }
   }
+
+  abstract static class XdsLoadReportClientFactory {
+
+    static XdsLoadReportClientFactory DEFAULT_INSTANCE = new XdsLoadReportClientFactory() {
```

Comment:
If you prefer the getter, then private the field.
Also `final`.

Reply:
Done.